pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceBaseArguments.java

```diff
@@ -133,6 +140,15 @@ public void fillArgumentsFromProperties() {
                     .getProperty("tlsEnableHostnameVerification", ""));
 
         }
+
+        if (proxyServiceURL == null) {
+            proxyServiceURL = prop.getProperty("proxyServiceURL");
+        }
+
+        if (proxyProtocol == null) {
+            proxyProtocol = ProxyProtocol.valueOf(prop.getProperty("proxyProtocol"));
```

Comment:
should I handle exception here if user provides some different option in config?